pkgs/by-name/ha/harlequin/package.nix

```
@@ -40,9 +60,18 @@ python3Packages.buildPythonApplication rec {
       questionary
       rich-click
       sqlfmt
-      textual
-      textual-fastdatatable
-      textual-textarea
+      textual_0_82
+      (textual-fastdatatable.override {
+        textual = textual_0_82;
+      })
+      (
+        (textual-textarea.override {
+          textual = textual_0_82;
+        }).overrideAttrs
+        (_: {
+          disabledTestPaths = [ "tests/functional_tests" ];
+        })
+      )
```

We can have a smaller diff, here:

```suggestion
      (textual-fastdatatable.override { textual = textual_0_82; })
      ((textual-textarea.override { textual = textual_0_82; }).overrideAttrs (_: {
        disabledTestPaths = [ "tests/functional_tests" ];
      }))
```

This is also more readable IMO.